.github/workflows/README.md

```diff
@@ -0,0 +1,53 @@
+# Inspektor Gadget CI Workflow
```

Comment:
I suppose this file should be moved to docs/ as done in https://github.com/kinvolk/inspektor-gadget/pull/561.

Reply:
Done